ui-tests/src/test/java/org/openmrs/reference/MergeVisitsTest.java

```diff
@@ -31,12 +30,11 @@
     @Before
     public void setUp() {
         patient = createTestPatient();
-        new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
```

Comment:
Thanks @HerbertYiga  Why this change ,what happens when you test it out before you touch it